andela/travel_tool_front
# #160343793 Fix missing destination column for mobile view

#### What does this PR do?
 Fixes missing destination column for mobile view

#### Description of Task to be completed?
- Add class to show destination field for mobile view


#### How should this be manually tested?
- Pull this branch
- Start the server using `make start`
- Start the backend using `make start`
- Open your browser and navigate to `http://travel-tool.andela.com:3000/requests`
- Change your view to tablet or mobile view
- You should also see the destination field in the requests table

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
 [#160343793](https://www.pivotaltracker.com/story/show/160343793)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-07 at 12 08 40 pm" src="https://user-images.githubusercontent.com/25195857/45215926-d6992780-b296-11e8-8ce7-d4cd5dd5d320.png">


#### Questions:
N/A
